bucket/download.py

```diff
@@ -94,13 +94,17 @@ def askForBucket():
 
 # Download training images.
 image_files = [image for image in annotations.keys()]
+image_files_exceptions = []
 for file in image_files:
```

Comment:
This was me, but can we change `file` to `f`, I think `file` is a reserved word

Reply:
I just copied the variable `image_files` and added `exception` at the end. What did you mean `file` is a reserved word?

Reply:
Maybe you meant
`for file in image_files:`